XiEditor/EditViewController.swift

```diff
@@ -505,19 +530,29 @@ class EditViewController: NSViewController, EditViewDataSource, FindDelegate, Sc
         }
         dragEvent = theEvent
     }
-    
+
     override func mouseUp(with theEvent: NSEvent) {
         dragTimer?.invalidate()
         dragTimer = nil
         dragEvent = nil
     }
-    
+
+    @objc func sendHover() {
+        if let event = hoverEvent {
+            let hoverPosition = editView.bufferPositionFromPoint(event.locationInWindow)
+            hoverTimer?.invalidate()
```

Comment:
Compilation is failing for me on these two lines. Looking more closely, I see it is not declared anywhere.

Reply:
Also, after removing these lines, hover does not work. Am I missing something?